src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calcuate = {
```

Comment:
```suggestion
  const calcuator = {
```
название переменной с объектом должно быть не в множественном числе